docs/developers/cosmwasm-contract-deployment.md

```diff
@@ -37,6 +37,16 @@ we want it to be as small as possible.
 The CosmWasm team provides a tool called `rust-optimizer` which we need
 [Docker](./cosmwasm-dependency.md/#docker-installation) for in order to compile.
 
+````mdx-code-block
```

Comment:
this won't work on a Markdown file. It would work on a .mdx file

Reply:
Interesting, it runs locally just fine. Adding change regardless. Thank you!

![2022-12-06 13 32 12](https://user-images.githubusercontent.com/46639943/205993266-e69f317c-6411-4033-897c-36d82146f01b.gif)